src/v0/destinations/adobe_analytics/networkHandler.js

```diff
@@ -0,0 +1,54 @@
+const { InstrumentationError } = require('@rudderstack/integrations-lib');
+const { proxyRequest, prepareProxyRequest } = require('../../../adapters/network');
+const {
+  processAxiosResponse,
+} = require('../../../adapters/utils/networkUtils');
+const { DESTINATION } = require('./config');
+
+
+/**
+ * Extract data inside different tags from an xml payload
+ * @param {*} xml 
+ * @param {*} tagName 
+ * @returns data inside the tagName
+ */
+function extractContent(xmlPayload, tagName) {
+  const pattern = new RegExp(`<${tagName}>(.*?)</${tagName}>`);
+  const match = xmlPayload.match(pattern);
+  return match ? match[1] : null;
+}
+
+const responseHandler = (destinationResponse, dest) => {
+  const message = `[${DESTINATION}] - Request Processed Successfully`;
+  const { response, status } = destinationResponse;
+
+  // Extract values between different tags
+  const responseStatus = extractContent(response, 'status');
+  const reason = extractContent(response, 'reason');
+
+  // if the status tag in XML contains FAILURE, we build and throw an explicit error
+  if (responseStatus === 'FAILURE') {
```

Comment:
What happens when resposnseStatus is undefined / null ?

Reply:
`resposnseStatus` is either `SUCCESS` or `FAILURE` only. [Reference](https://developer.adobe.com/analytics-apis/docs/1.4/guides/data-insertion/troubleshooting/)